### PR TITLE
(editorial) minor punctuation and whitespace edits

### PR DESCRIPTION
**Pull request by TallTed**

### CODE REVIEW

**Comment by mprorock** [comment]
@TallTed holding on this one due to issue #80 and until we merge the related PR #81 as it is going to be a pretty good overhaul and will likely cause some breakage. as soon as those are in will get this in.

**Comment by TallTed** [comment]
@mprorock - This PR is tiny, and will cause minimal merge conflicts with any other PRs.  #81 is pretty huge, and will cause substantial merge conflicts with all other PRs.  The choice that will cause the least additional work would be to merge this and any other small PRs first.

**Comment by mprorock** [comment]
> The choice that will cause the least additional work would be to merge this and any other small PRs first.
@OR13 thoughts?  my concern is getting the json-ld validating correctly first and foremost, then merging other changes against that.

**Comment by OR13** [comment]
@mprorock looks like you may have forgot to comment.. at any rate, I appreciate having the IRI change merge first, since it impacts signatures and credentials... hopefully this change set can be updated / revised.

**Comment by mprorock** [comment]
@TallTed can you verify that i merged your changes as desired?  

@OR13 is my understanding that the json-ld portions of index should not be edited directly correct?  i believe that is auto-generated

**Comment by TallTed** [comment]
@OR13 - @mprorock's comment got caught in a quotation line... His comment was:
> @OR13 thoughts? my concern is getting the json-ld validating correctly first and foremost, then merging other changes against that.

@mprorock -- There're now several difference between my branch and the base branch that I didn't author.  I think it would be best, if possible, to back out your merge into/from my branch, and let me handle the merge conflicts ... which _may_ mean I drop this PR, and create a fresh one.

**Comment by mprorock** [comment]
@TallTed totally fine with you backing that commit out.  might be cleanest just to copy over your top few changes to the latest main.  if you close and reopen a new pull i will be watching for it and will get it accepted quickly

**Review by OR13 — APPROVED** [review]

File: docs/index.html [hunk 1]
```diff
@@ -3646,21 +3645,13 @@ <h2>Vocabulary</h2>
...
-              "links": {
-                "relationship-records": "https://api.gleif.example.org/api/v1/lei-records/8W58S473UJBX3RDKQDJN/direct-child-relationship",
-                "related": "https://api.gleif.example.org/api/v1/lei-records/8W58S473UJBX3RDKQDJN/direct-children"
-              }
-            },
-            "isins": {
-              "links": {
-                "related": "https://api.gleif.example.org/api/v1/lei-records/8W58S473UJBX3RDKQDJN/isins"
-              }
+            "https://schema.org/unitCode": {
```
Comment (OR13) [inline docs/index.html:3648]: I think this change is likely to get overwritten every time the spec is regenerated.

Comment (OR13) [reply docs/index.html:3648]: We generate the JSON, embed it in respec, and then render respec at runtime via javascript.

**Comment by TallTed** [comment]
replaced by #92

**Pull request closed without merging**